spec/features/cancel_spec.rb

```diff
@@ -1,13 +1,9 @@
 describe "Canceling a request" do
-  let (:client_data) { FactoryGirl.create(:ncr_work_order) }
-  let (:proposal) { FactoryGirl.create(:proposal, :with_approver, client_data_type: "Ncr::WorkOrder", client_data_id: client_data.id) }
-  let (:user) { FactoryGirl.create(:user, id: 123456) }
-
-  before do
-    login_as(proposal.requester)
-  end
+  let!(:client_data) { FactoryGirl.create(:ncr_work_order, :with_approver) }
+  let!(:proposal) { client_data.proposal }
 
   it "shows a cancel link for the requester" do
+    login_as(proposal.requester)
```

Comment:
Are you sure this is needed (and that the factory fix above didn't resolve)? It seems wrong to repeat this in every test